plasticos_geolocalize/models/res_partner_geo.py

```
@@ -0,0 +1,115 @@
+        partners = self.search(domain, limit=BATCH_SIZE)
+        if not partners:
+            _logger.info("Geo backfill: no partners need geocoding.")
+            return
+
+        success = 0
+        for partner in partners:
+            try:
+                partner.geo_localize()
+                if partner.partner_latitude:
+                    success += 1
+            except Exception:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)